Dev/VideoChatClient/Input/IInput.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <cstdint>
+#include <vector>
+
+typedef std::vector<uint8_t> Buffer;
```

Comment:
use Using instead of typedef, race the Resharper!!🐱‍🏍

Reply:
Can't! I tried 😥